client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -0,0 +1,164 @@
+import { isBusiness, isEcommerce, isEnterprise } from '@automattic/calypso-products';
+import { Button, Dialog } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import EligibilityWarnings from 'calypso/blocks/eligibility-warnings';
+import { userCan } from 'calypso/lib/site/utils';
+import { isCompatiblePlugin } from 'calypso/my-sites/plugins/plugin-compatibility';
+import { recordGoogleEvent } from 'calypso/state/analytics/actions';
+import {
+	getEligibility,
+	isEligibleForAutomatedTransfer,
+} from 'calypso/state/automated-transfer/selectors';
+import { isRequestingForSites } from 'calypso/state/plugins/installed/selectors';
+import { removePluginStatuses } from 'calypso/state/plugins/installed/status/actions';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { default as checkVipSite } from 'calypso/state/selectors/is-vip-site';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import './style.scss';
+
+const PluginDetailsCTA = ( props ) => {
+	const translate = useTranslate();
+	const { pluginSlug, selectedSite, isPluginInstalledOnsite, siteIds } = props;
+
+	const requestingPluginsForSites = useSelector( ( state ) =>
+		isRequestingForSites( state, siteIds )
+	);
+
+	// Site type
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, selectedSite?.ID ) );
+	const isVip = useSelector( ( state ) => checkVipSite( state, selectedSite?.ID ) );
+	const isAtomic = useSelector( ( state ) => isSiteAutomatedTransfer( state, selectedSite?.ID ) );
+	const isJetpackSelfHosted = selectedSite && isJetpack && ! isAtomic;
+
+	// Eligibilities for Simple Sites.
+	const { eligibilityHolds, eligibilityWarnings } = useSelector( ( state ) =>
+		getEligibility( state, selectedSite?.ID )
+	);
+	const isEligible = useSelector( ( state ) =>
+		isEligibleForAutomatedTransfer( state, selectedSite?.ID )
+	);
+	const hasEligibilityMessages =
+		! isJetpack && ( eligibilityHolds || eligibilityWarnings || isEligible );
+
+	if ( requestingPluginsForSites ) {
+		// Display nothing if we are still requesting the plugin status.
+		return null;
+	}
+	if ( ! isJetpackSelfHosted && ! isCompatiblePlugin( pluginSlug ) ) {
+		// Check for WordPress.com compatibility.
+		return null;
+	}
+
+	if ( ! selectedSite || ! userCan( 'manage_options', selectedSite ) ) {
+		// Check if user can manage plugins.
+		return null;
+	}
+
+	if ( isPluginInstalledOnsite ) {
+		// Check if already instlaled on the site
+		return null;
+	}
+
+	return (
+		<div className="plugin-details-CTA__container plugin-details__header">
```

Comment:
Should we move the CSS from `plugin-details__header` to `plugin-details-CTA__container` and remove the `plugin-details__header` class from here?

Reply:
I decided to keep the `plugin-details__header` on PluginDetailsCTA and PluginDetailsHeader to avoid the duplication of CSS code. I found the better place for this class to be is PluginDetails because it also applies to the placeholders. Do you have any other idea of how can we avoid this duplication?

Reply:
In fact, I have later understood that this class is reused in other components (eg placeholder) which in fact need to exhibit the same behavior. Seems good for now.

Reply:
I have moved the CSS rules to a `variable` file to be reused in both places. 
This way I was able to remove the `plugin-details__header` and not duplicate the code. 

c70a0f26be50b0412179fc5173684238023b6dde